src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/CompilerPass/RegisterDatePointTypePassTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Tests\DependencyInjection\CompilerPass;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Doctrine\DependencyInjection\CompilerPass\RegisterDatePointTypePass;
+use Symfony\Bridge\Doctrine\Types\DatePointType;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class RegisterDatePointTypePassTest extends TestCase
+{
+    public function testRegistered()
+    {
+        $container = new ContainerBuilder();
+        $container->setParameter('doctrine.dbal.connection_factory.types', ['foo' => 'bar']);
+        (new RegisterDatePointTypePass())->process($container);
+
+        $expected = [
+            'foo' => 'bar',
+            'date_point' => ['class' => DatePointType::class],
+        ];
+        $this->assertSame($expected, $container->getParameter('doctrine.dbal.connection_factory.types'));
+    }
+
+    public function testRegisteredDontFail()
+    {
+        $container = new ContainerBuilder();
+        (new RegisterDatePointTypePass())->process($container);
+
+        $this->expectNotToPerformAssertions();
```

Comment:
Why do we have this test (i.e. what is the goal to achieve here)?

Reply:
Because we have the same test in the RegisterUidTypePassTest.php

Reply:
I would remove it. I don't see the value in having it.

Reply:
removed